### PR TITLE
Fix typos in SEM estimator

### LINKED ISSUE #1803: Bug in estimators/SEMEstimator.py
### Subject of the issue

Potential bug in the SEM script

### Bugs

* Following line has same variable for `zeta` and `B`, key typo: 
```
https://github.com/pgmpy/pgmpy/blob/2c10319adc66327ec57d0f247b6a07a10be39a23/pgmpy/estimators/SEMEstimator.py#L371
```

* In the following line, `S` is a Tensor so an error is thrown. Should be `sample_cov` or `S.detach().numpy()`
```
https://github.com/pgmpy/pgmpy/blob/2c10319adc66327ec57d0f247b6a07a10be39a23/pgmpy/estimators/SEMEstimator.py#L342
```

### PR DESCRIPTION
**Pull request by jihyeseo**
### Your checklist for this pull request
Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the **dev branch** (left side). Also you should start *your branch* off *our dev*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.

### Issue number(s) that this pull request fixes
- Fixes #1803 

### List of changes to the codebase in this pull request
- 
-
-

### CODE REVIEW

**Comment by ankurankan** [comment]
Thanks for the PR.
